### DISCUSSION TITLE
Does UTMStack have centralized configuration for agents?

### DISCUSSION THREAD

**Discussion by tssingtampa** [utmstack/UTMStack#487] [Q&A] [answered]
Hello community, I would like to know if UTMStack has centralized agent configurations to enable or disable properties or functions?

**Comment by c3s4rfred** [comment]
Hi @tssingtampa, no, it doesn't, the agent is a component with particular configurations that appear in the corresponding UTMStack integration. The only settings needed for configuration are the integrations enabled to receive logs; no other configuration is needed at the moment.

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @tssingtampa, after a deep review, we find that we already have a feature about this in our roadmap, so, in future versions, we will have a graphic agent configuration.
